src/libraries/System.Speech/src/Resources/Strings.resx

```diff
@@ -1157,7 +1157,43 @@ Possible constructors are: '{2}'.</value>
     <value>The local recognizer cannot activate a rule, dictation, or other stream until it is initialized. Call SetInput first.</value>
   </data>
   <data name="SapiErrorUnused0" xml:space="preserve">
-    <value>Error code not used.</value>
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused1" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused2" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused3" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused4" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused5" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused6" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused7" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused8" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused9" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused10" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused11" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
+  </data>
+  <data name="SapiErrorUnused12" xml:space="preserve">
+    <value></value> <!-- SAPI does not use this code -->
```

Comment:
Why are all of these entries needed? What breaks if they're missing?

Reply:
If SAPI actually returns an error code that it claims not to (that doesn't have a define in sperror.h) then this will ensure we don't throw an exception. Defensive, I guess.

Reply:
Where would such an exception be thrown?

Reply:
Where it tries to load the resource to put it in an exception, right?

Reply:
It's using SR.GetResourceString?  Doesn't that just return null if it can't find the resource string?

Reply:
Hmm, for the "standard" SR system, GetResourceString can return null, but it looks like we always generate a default value. TIL that Exception and ?many of its derived classes accept a null message and use a default one.

In the system that Speech uses, it also uses a default message:
https://github.com/danmosemsft/runtime/blob/e27468c01ed805df44d17e6a632f474bf533d4a3/src/libraries/System.Speech/src/SR.cs#L18

At this point I am going to merge as is unless you object, as it seems a lower risk way to get it into the next months release. I will follow up in master though with cleanup to remove these placeholder resources.